src/main/java/nextstep/member/exception/UnAuthorizedException.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.member.exception;
+
+public class UnAuthorizedException extends RuntimeException{
```

Comment:
- 전체적으로 예외 클래스들에 사용되지 않는 생성자들이 많은데, 사용하지 않는다면 제거해보면 어떨까요?
- 코드 포맷팅 확인해주시면 좋을 것 같아요.
```suggestion
public class UnAuthorizedException extends RuntimeException {
```